client/src/components/EventEdit/Header/index.js

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import Moment from 'react-moment';
+import PropTypes from "prop-types";
+import { FiPrinter, FiSettings, FiDownload } from "react-icons/fi";
+import styled from "styled-components";
+
+import IconButton from "../../Common/IconButton";
+import {
+  theme1,
+  fontSize64,
+  fontSize24,
+  spacing24,
+} from "../../../resources/style-constants";
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  background-color: ${props => props.bgcolor || theme1};
+`;
+
+const Spacer = styled.div`
+  height: ${spacing24};
+`;
+
+const TitleContainer = styled.h1`
+  font-size: ${fontSize64};
+  margin: 0;
+`;
+
+const DateContainer = styled.h5`
+  font-size: ${fontSize24};
+  opacity: 0.5;
+  margin: 0;
+`;
+
+const ButtonBarContainer = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+/**
+ * The header which appears at the top of the event-edit page.
+ */
+const Header = ({
+  title,
+  startDate,
+  // endDate, TODO: do we even need an end date? Leaving it out
+  // for now because it just adds complexity.
+  backgroundColor,
+  onPressDownload,
+  onPressPrint,
+  onPressSettings,
+  ...props
+}) => {
+
+  return (
+    <Container {...props}>
```

Comment:
No need to update this, but it might be useful for future tasks. I created an element called Stack. You can pass a gapSize to it, and all of the child elements will be spaced out gapSize apart instead of having to use Spacer.

Reply:
Oh perfect! I'll switch to this